test/testBtc.ts

```diff
@@ -0,0 +1,327 @@
+    expect(process.env.DEVICE_ID).to.not.equal(null);
+    await helpers.connect(client, process.env.DEVICE_ID);
+    expect(client.isPaired).to.equal(true);
+    expect(client.hasActiveWallet()).to.equal(true);
+    activeWalletUID = helpers.copyBuffer(client.getActiveWallet().uid);
+  });
+});
+
+describe('exportSeed', () => {
+  it('Should get GP_SUCCESS for a known, connected wallet', async () => {
+    expect(activeWalletUID).to.not.equal(null, 'No wallet found');
+    const jobType = helpers.jobTypes.WALLET_JOB_EXPORT_SEED;
+    const jobData = {};
+    const jobReq = {
+      testID: 0, // wallet_job test ID
+      payload: helpers.serializeJobData(jobType, activeWalletUID, jobData),
+    };
+
+    const res = await helpers.execute(client, 'test', jobReq);
+    const _res = helpers.parseWalletJobResp(res, client.fwVersion);
+    expect(_res.resultStatus).to.equal(0);
+    const data = helpers.deserializeExportSeedJobResult(_res.result);
+    const activeWalletSeed = helpers.copyBuffer(data.seed);
+    wallet = bip32.fromSeed(activeWalletSeed);
+  });
+});
+
+async function run(p) {
+  await testSign(p.txReq, p.signingKeys, p.sigHashes);
+}
+
+async function runTestSet(opts, wallet, inputsSlice, next) {
+  if (TEST_TESTNET) {
+    // Testnet + change
+    try {
+      opts.isTestnet = true;
+      opts.useChange = true;
+      await run(
+        helpers.setup_btc_sig_test(opts, wallet, inputsSlice, prng)
+      );
+    } catch (err) {
+      expect(err).to.equal(
+        null,
+        `Failed in (testnet, change): ${err.message()}`
+      );
+      continueTests = false;
+      next(err);
+    }
+    // Testnet + no change
+    try {
+      opts.isTestnet = true;
+      opts.useChange = false;
+      await run(
+        helpers.setup_btc_sig_test(opts, wallet, inputsSlice, prng)
+      );
+    } catch (err) {
+      expect(err).to.equal(
+        null,
+        `Failed in (testnet, !change): ${err.message()}`
+      );
+      continueTests = false;
+      next(err);
+    }
+  }
+  // Mainnet + change
+  try {
+    opts.isTestnet = false;
+    opts.useChange = true;
+    await run(
+      helpers.setup_btc_sig_test(opts, wallet, inputsSlice, prng)
+    );
+  } catch (err) {
+    expect(err).to.equal(
+      null,
+      `Failed in (!testnet, change): ${err.message()}`
+    );
+    continueTests = false;
+    next(err);
+  }
+  // Mainnet + no change
+  try {
+    opts.isTestnet = false;
+    opts.useChange = false;
+    await run(
+      helpers.setup_btc_sig_test(opts, wallet, inputsSlice, prng)
+    );
+  } catch (err) {
+    expect(err).to.equal(
+      null,
+      `Failed in (!testnet, !change): ${err.message()}`
+    );
+    continueTests = false;
+    next(err);
+  }
+  next();
+}
+
+describe('Test segwit spender (p2wpkh)', function () {
+  beforeEach(() => {
+    expect(continueTests).to.equal(true, 'Previous test failed. Aborting');
+  });
+  //@ts-expect-error - it.each is not included in @types/mocha
```

Comment:
Btw I am open to switching out this module for something TS-friendly. I have always **hated** the API of `it.each`. Doesn't need to be done now.